src/JuMPContainer.jl

```diff
@@ -99,6 +99,7 @@ end
 
 pushmeta!(x::JuMPContainer, sym::Symbol, val) = (x.meta[sym] = val)
 getmeta(x::JuMPContainer, sym::Symbol) = x.meta[sym]
+hasmeta(x::JuMPContainer, sym::Symbol) = haskey(x.meta, sym)
```

Comment:
I have addressed this in [this branch](https://github.com/blegat/JuMP.jl/tree/meta). Once this PR is merged, I will rebase it and do a PR